cc.arduino.arduinoide2.json

```diff
@@ -0,0 +1,119 @@
+{
+    "app-id": "cc.arduino.arduinoide2",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "arduino-ide",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
```

Comment:
why do you need system-bus?

Reply:
probably not. I will remove it for now. If we need it in the future it can be added again

Reply:
no. 

What should be done is determine which name are needed if any. Unless this is a tool to explore d-bus (I don't think so)

Reply:
ok so i was a little hasty with removing dbus. i have decided to keep the dbus permission because removing it creates the error: `[3:0521/020426.434313:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory` when the application is run. So i'd say it's safe to say it is required by a part of the application